proposals/0167-app-services.md

```diff
@@ -305,25 +302,43 @@ A navigation service is defined as a service that is currently listed as the nav
         <param name="nextInstructionDistanceScale" type="Float" mandatory="false">
             <description>Distance till next maneuver (starting from) from previous maneuver.</description>
         </param>
-	
+        
+        <param name="prompt" type="String" mandatory="false">
+            <description>This is a prompt message that should be conveyed to the user through either display or voice (TTS). This param will change often as it should represent the following: approaching instruction, post instruction, alerts that affect the current navigation session, etc.</description>
+        </param>
     </struct>
 
-    <struct name="NavInstruction">
+    <struct name="NavigationInstruction">
 	    <param name="locationDetails" type="LocationDetails" mandatory="true">
 	    </param>
 	
-	    <param name="action" type="NavAction" mandatory="true">
+	    <param name="action" type="NavigationAction" mandatory="true">
+	    </param>
+	    
+	    <param name="eta" type="DateTime" mandatory="false">
 	    </param>	
 	
-	    <param name="direction" type="int" minValue="0" maxValue="359" mandatory="false">
-	        <description>The angle at which this instruction takes place. For example, 0 would mean straight, 45 a sharp right, 180 a U-Turn, etc. </description>
+	    <param name="bearing" type="int" minValue="0" maxValue="359" mandatory="false">
+	        <description>The angle at which this instruction takes place. For example, 0 would mean straight, <=45 is bearing right, >= 135 is sharp right, between 45 and 135 is a regular right, and 180 is a U-Turn, etc. </description>
 	    </param>
 	
-	    <param name="junctionType" type="NavJunctionType" mandatory="false">
+	    <param name="junctionType" type="NavigationJunction" mandatory="false">
 	    </param>	
+	    	
+	    <param name="drivingSide" type="Direction" mandatory="false">
+	        <description>Used to infer which side of the road this instruction takes place. For a U-Turn (Action=Turn, direction=180) this will determine which direction the turn should take place. </description>
+	    </param>
+	    
+	    <param name="details" type="String" mandatory="false">
+	        <description>This is a string representation of this instruction. For use to display instruction to users. This is not intended to be read aloud to users, see the param prompt in NavigationServiceData for that.</description>
```

Comment:
Update to:	        
<description>This is a string representation of this **instruction, used to display instructions to the users.** This is not intended to be read aloud to **the** users, see the param prompt in NavigationServiceData for that.</description>